packages/translations/scripts/extract-string.js

```diff
@@ -6,8 +6,10 @@ const getRegexPattern = () => /(i18n_default_text={?|localize\()\s*(['"])\s*(.*?
 
 const getStringsFromInput = (input, i18n_marker = getRegexPattern()) => {
     const messages = [];
+    let result;
+    const newResult = (result = i18n_marker.exec(input));
```

Comment:
i can't understand this
why don't you assign the value to result variable directly?

Reply:
You are right. I'll fix this